holoscribe/Cargo.toml

```diff
@@ -8,5 +8,7 @@ edition = "2021"
 [dependencies]
 clap = { version = "4.1.6", features = ["derive"] }
 obj = "0.10.2"
+regex = "1.7.1"
```

Comment:
From what I understand, we should be able to safely use `regex = "1"` for crates that are > 1.0. Allows end-users to use updated versions of crates without (in theory) breaking backwards compatibility.